FrameworkSystem/DB/InstalledComponentsDB.py

```diff
@@ -244,7 +244,7 @@ class HostLogging( componentsBase ):
   hostName = Column( 'HostName', String( 32 ), nullable = False, primary_key = True )
   # status
   DIRAC = Column( 'DIRACVersion', String( 32 ) )
-  Extension = Column( 'Extension', String( 64 ) )
+  Extensions = Column( 'Extensions', String( 64 ) )
```

Comment:
We should not force anyway to make this change in a patch. Please revert or make it backward compatible.